bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -106,14 +106,17 @@ export default function addMeeting(meeting) {
   const meetingEnded = false;
 
   let { welcomeMsg } = newMeeting.welcomeProp;
-  const sanitizedWelcomeText = SanitizeHTML(welcomeMsg, {
-    allowedTags: ['b', 'strong', 'i', 'u', 'a', 'br', 'img'],
+
+  const sanitizeTextInChat = original => SanitizeHTML(original, {
+    allowedTags: ['a', 'b', 'br', 'i', 'img', 'li', 'small', 'span', 'strong', 'u', 'ul'],
```

Comment:
Just wondering: If per default non-inline elements like `ul` are allowed, shouldn't be `p` and `div` allowed as well?